v3/tasks/20250214-v3-protocol-fee-controller-v2/readme.md

```diff
@@ -6,5 +6,5 @@ There is nothing to deprecate, as the original Protocol Fee Controller was deplo
 
 ## Useful Files
 
-- [Code](https://github.com/balancer/balancer-v3-monorepo/commit/0c38dc629386c401e3d6094981d7990f2fe0a379)
+- [Code](https://github.com/balancer/balancer-v3-monorepo/commit/37c26dc46cc466fecf29b4650408db93f060b778)
```

Comment:
I'm getting a difference with the build info at this commit

Reply:
Yes; that's why I set it to draft. It's still pending changes from the monorepo, so there's no commit yet. I just updated it with the latest from that branch and propagated it through to be absolutely sure the changes didn't affect the migration